CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute
+
+## Rights
+The license (see LICENSE) applies to all contributions.
+
+## Issue Conventions
+When submitting bugs, please filll out the bug report template in full.
+
+Please search existing issues, open and closed, before creating a new one.
+
+## Git conventions
+Work on features should be made on a fork of `geoutils` and submitted as a pull request (PR) to main or a relevant branch.
```

Comment:
Great! Maybe just change "main" to "master" here to be consistent with the primary branch name.

Reply:
Fixed this in 1e0c370